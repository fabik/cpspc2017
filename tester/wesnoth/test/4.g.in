90000 51
1
45879
