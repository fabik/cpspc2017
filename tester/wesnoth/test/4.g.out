45879 90000 0.5570064019
